src/Elastic.CommonSchema.NLog/EcsLayout.cs

```diff
@@ -21,6 +23,9 @@ public class EcsLayout : Layout
 	{
 		public const string Name = nameof(EcsLayout);
 
+		private static readonly bool NLogApmLoaded =
```

Comment:
Think it should be a static-method instead of being static-member-variable that is initialized on class-type-initiailization. It is recommended to reduce the amount of logic being execute on class-type-initialization, and instead perform the logic in class-object-creation.

Reply:
++ will amend 👍